tests/framework/PostProcessors/Validation/test_validation_gate_pcm.xml

```diff
@@ -0,0 +1,151 @@
+<?xml version="1.0" ?>
+<Simulation verbosity="all">
+  <RunInfo>
+    <WorkingDir>PhysicsGuidedCoverageMapping</WorkingDir>
+    <Sequence>mcRunExp,mcRunApp,mcRunMsr,PCM</Sequence>
+    <batchSize>1</batchSize>
+  </RunInfo>
+
+  <TestInfo>
+    <name>framework/PostProcessors/Validation/test_validation_gate_pcm</name>
+    <author>dhuang</author>
+    <created>2021-07-21</created>
+    <classesTested>PostProcessors.Validation</classesTested>
+    <description>
+        This test is aimed to show how PCM works.
+        For simplicity, this test is using attenuate as experiment and tensor_ploy as application.
+        The output name convention is ...``feature name''\_``target name''\_``metric name''.
```

Comment:
Update this comments to reflect the model and the outputs of PCM.

Reply:
Updated.